Lib/fontTools/feaLib/ast.py

```diff
@@ -62,6 +66,9 @@
     "SingleSubstStatement",
     "SizeParameters",
     "Statement",
+    "STATAxisValueRecord",
+    "STATDesignAxis",
```

Comment:
you'll notice the convention in feaLib.ast module is to name the classes with an ending "*Statement" when they consist of a "statement" (i.e. a line of FEA ending with semicolon). It may be good to keep that convention for consistency. All the new classes defined here are statements in this sense. Mind renaming them, even though the names becomes a bit long?